.github/workflows/bors.yml

```diff
@@ -248,7 +248,10 @@ jobs:
         uses: liskin/gh-problem-matcher-wrap@v3
         with:
           linters: gcc
-          run: lake test
+          run: |
+            # make sure everything is available for import_all.lean
```

Comment:
There's no file `import_all.lean` in mathlib (and I don't find the relation to `mk_all` obvious; that doesn't require building batteries). Would you like to clarify this comment?
(The same applies to the three other copies of this comment.)

Reply:
`tests/import_all.lean`. I'll update the comment.

Reply:
:bulb: Thanks!